adoptingerlang.org

```diff
@@ -4101,20 +4101,118 @@ With these broad lines covered, we can start exploring the processes and practic
+
+*** A Production Language
+
+As you iterate over all of these criteria you have established, you should keep some end-goal in mind. For example, you might want to end up with the following things covered (note that many of these are covered in this text to make things easier for more Erlang users out there):
+
+- How to install and set-up your development environment with all the versions required
+- A blessed list of libraries that have been validated and that most projects are using
+- The procedures around starting new projects (setting tests up, which linters to use, how to get CI/CD going, templates, and so on)
+- The places where you can get help (such as internal mailing lists, chatrooms, or the name of people to contact)
+- A library of learning resources, which can be books, tutorials, videos, or internal documents you found useful
+- Security guidelines
+- Checklists of things to do to bring a thing to production grade
+- How to benchmark your code
+- How to instrument your code
+- Architecture and contribution guidelines
+- Runbooks and playbooks for common problems
+- Interview questions
+
+Not all of these need to be expansive tutorials in wikis; some can be accomplished by pairing, lunch and learns, presentations, and so on. Do note that regardless of how you choose to help on-board people for these items, the developers you hire or move across teams already have to figure _all of these_ out already, many without explicit support. This invisible work is being done all the time. The aim is to make it visible.
+
+In choosing to support polyglot languages and surfacing what your team believes is necessary to help them move from one project to the next (without creating orphaned technology), you're likely to actually make switching between teams easier than if you just had one programming language without that support. If you really want to help make such principles part of your culture, show that you value teams that participate in reaching and improving standard practices, and who do work to bring existing codebases up to the desired level of excellence. It can be tricky to get enough buy-in to progressively build these resources and keep them alive, so that last bit is particularly important.
+
+In short, let's focus on how people learn these things and help them do it, rather than pretending they already know everything. Prepare starter guides around domain knowledge. Map out areas where you are currently under-serving your engineers and in which you possibly need to invest. Establish ways to socially keep language-specific knowledge and tooling alive. And possibly discover factors that you did not know were important but turn out to be critical to your teams doing their work.
```

Comment:
```suggestion
In short, let's focus on how people learn and help them do it, rather than pretending they already know everything. Prepare starter guides around domain knowledge. Map out areas where you are currently under-serving your engineers and in which you possibly need to invest. Establish ways to socially keep language-specific knowledge and tooling alive. And possibly discover factors that you did not know were important but turn out to be critical to your teams doing their work.
```

Just a suggestion. I think it reads better than "things".